process.c

```diff
@@ -3591,7 +3593,7 @@ save_env(struct rb_execarg *sargp)
     if (!sargp)
         return;
     if (sargp->env_modification == Qfalse) {
-        VALUE env = rb_const_get(rb_cObject, id_ENV);
+        VALUE env = rb_envtbl();
```

Comment:
I think it would make more sense to call this `rb_process_env`.

Reply:
I'm going to merge this.  Feel free to send a PR to rename the function if you want.

Reply:
Okay 👍